tests/test_non_ascii_search_string/code/dot_c.c

```diff
@@ -0,0 +1 @@
+// Just a .c file to test find-by-extension
```

Comment:
Instead of copying all of test_basic, can you cut it down to the minimum required to test the fix?